webdriver/pom.xml

```diff
@@ -37,7 +37,7 @@
     </distributionManagement>
 
     <properties>
-        <selenium.version>2.47.0</selenium.version>
+        <selenium.version>2.49.1</selenium.version>
```

Comment:
May I ask you why or where I can read about it? I didn't want to modify pom, but selenium 2.47.0 "dislikes" modern versions of firefox(43.0.4) and phantomjs(2.0+). It just doesn't work at all, i.e. it  starts a browser up and stay stalled. Updating selenium solved that problem. that's why I did this